packages/jsii-calc-base-of-base/test/assembly.jsii

```diff
@@ -16,7 +16,7 @@
     "type": "git",
     "url": "https://github.com/awslabs/jsii.git"
   },
-  "schema": "jsii/1.0",
+  "schema": "jsii/0.9.0",
```

Comment:
Waitaminit! I don't like how this is version number is going backwards, and we already have a `jsiiVersion` field that encodes the JSII version. Why can't we just discretely bump the schema versions 1 -> 2 -> 3 etc? Saves us having to do the whole "what's the next release version" dance, too.

Reply:
I don't really care. I kinda felt it was nice to correlate that with the `jsii` version so you can, from an error (like "Whoosh the version is `jsii/0.9.0`, but I expected to find `jsii/0.42.12`), determine what you need to do. I however would only update that version number if/when the spec gets a breaking change (so it acts as a "minimum required version" of sorts.